var/spack/repos/builtin/packages/bash-completion/package.py

```diff
@@ -42,6 +42,10 @@ class BashCompletion(AutotoolsPackage):
     # Other dependencies
     depends_on('bash@4.1:', type='run')
 
+    def install(self, spec, prefix):
+        mkdirp(join_path(prefix.share, 'bash-completion', 'completions'))
+        return super(BashCompletion, self).install(spec, prefix)
```

Comment:
Another way to solve this that might be more Spacky and less Pythonic would be:
```python
@run_before('install')
def create_install_directory(self):
    mkdirp(join_path(self.prefix.share, 'bash-completion', 'completions'))
```
Still, I'm surprised their Makefile doesn't already do this.